src/time/mktime.c

```diff
@@ -0,0 +1,11 @@
+#include "time_impl.h"
+#include <errno.h>
+
+time_t mktime(struct tm *tm)
+{
+   struct tm new;
```

Comment:
Can you clean this up a bit. Remove the unused variables, this function ultimately becomes 
```c++
time_t mktime(struct tm* tm) {
   return __tm_to_secs(tm);
}

Reply:
Done